bin/c-kc-ngrok.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('commander');
+const { exec } = require('shelljs');
+const { formatProjectPrefix } = require('./lib/c-project');
+const { loadConfig, reportError } = require('./lib/c');
+
+const location = 'ngrok';
+
+program
+    .description('Expose services to the world via Ngrok.')
+    .option('-s', 'Stops and removes the ngrok pod.')
```

Comment:
@smebberson Do you think this would be better as the full `--stop` instead of `-s`? Just a minor thing, but `-s` could stand for start or stop.

Reply:
I think removing the option and adding a new command `c kc ngrok start` and `c kc ngrok stop` might also work too?